ZelBack/src/routes.js

```diff
@@ -1325,8 +1325,11 @@ module.exports = (app, expressWs) => {
   });
 
   // communication between multiple flux solution is on this:
-  app.ws('/ws/flux', (ws, req) => {
-    fluxCommunication.handleIncomingConnection(ws, req, expressWs.getWss('/ws/flux'));
+  app.ws('/ws/flux/:port', (ws, req) => {
```

Comment:
shouldnt be /ws/flux/:port?
Was this properly tested that the query/params exists on the websocket connection?

Reply:
if you check bellow, you have this 3 methods that work exactly the same way.
// WebSockets PUBLIC
  app.ws('/ws/id/:loginphrase', (ws, req) => {
    idService.wsRespondLoginPhrase(ws, req);
  });
  app.ws('/ws/zelid/:loginphrase', (ws, req) => { // DEPRECATED
    idService.wsRespondLoginPhrase(ws, req);
  });
  app.ws('/ws/sign/:message', (ws, req) => {
    idService.wsRespondSignature(ws, req);
  });

Reply:
tested in development nodes: https://github.com/RunOnFlux/flux/commit/4eb8b9f67afba44a3d1a670d7cf6d4eca9b5bb64

only route available on the nodes was:
  // communication between multiple flux solution is on this:
  app.ws('/ws/flux/:port?', (ws, req) => {
    fluxCommunication.handleIncomingConnection(ws, req, expressWs.getWss('/ws/flux/:port?'));
  });

as I removed the other on a previous commit and I only got incoming connections from the development nodes as it was expected.
![image](https://github.com/RunOnFlux/flux/assets/42519726/70f610d1-985b-470b-bf01-370ac4dbccfb)